src/OpenLoco/src/Objects/VehicleObject.cpp

```diff
@@ -586,7 +587,10 @@ namespace OpenLoco
         }
 
         // Verify we haven't overshot any lengths
-        assert(imgRes.imageOffset + offset == ObjectManager::getTotalNumImages());
+        if (imgRes.imageOffset + offset != ObjectManager::getTotalNumImages())
+        {
+            Console::logVerbose("Incorrect number of images for object");
```

Comment:
Is it very common for objects to have the wrong image count? In that case, it makes sense to go with `logVerbose`. If not, I think we should actually advertise it.

Regardless, is it possible to easily include the object name for reference?

Reply:
MAILSW4 suffers from it.

Reply:
There doesn't seem to be a huge amount that suffer from it but i did come across a few more when scrolling through the object selection window.